stdlib/InteractiveUtils/src/codeview.jl

```diff
@@ -1,17 +1,46 @@
 # This file is a part of Julia. License is MIT: https://julialang.org/license
 
+# highlighting settings
+highlighting = Dict{Symbol, Bool}(
```

Comment:
should these dictionaries be `const`?

Reply:
At the moment, there is no public API to change the configuration, so I think specifying `const` would only make them less convenient.

Of course, I'm in favor of adding the public API. However, it should be designed to be consistent with the design for customization APIs of other features (e.g. stack traces). I think that requires a separate discussion.